### DISCUSSION TITLE
How do I place two buttons one onto of each other in a single container?

### DISCUSSION THREAD

**Discussion by marianina8** [mum4k/termdash#341] [Q&A] [answered]
It seems like the buttons overlap each other because only one button shows up in the container when I use this code:
```go
    container.Left(
	    container.PlaceWidget(button1),
	    container.PlaceWidget(button2),
    ),
```
I've played around with margins and padding to see if I need to set some space between them...

**Comment by mum4k** [comment, marked as answer]
Thanks for reaching out @marianina8. The `Termdash` infrastructure only supports a single widget in each container. If you need to place multiple widgets, you will need to split the container into multiple containers first. There is some documentation on how the container layout works here:

https://github.com/mum4k/termdash/wiki/Grid-layout

**Reply by marianina8** [reply]
Thanks @mum4k !
